### ISSUE TITLE
Electrician New Hero - Model/Animation/Icons | 150€ Bounty

### ISSUE THREAD

**Issue by Shusama** [OpenAngelArena/oaa#2091]
# Hero: Electrician

### See also:
Electician New Hero - Code #2089 
Electician New Hero - Particles #2092
Electician New Hero - Sound #2090 

[General Guidelines for Model Issues](https://docs.google.com/document/d/15VtqjOfyL8aZiK7ufJp-G-rPkVGza-K3IkBHrMrLN-U/edit)
[Bug Bounty Guideline](https://docs.google.com/document/d/1Tpm0j1ekVc6X-LRlF_6Q81RMGVlOOO5EBvSf_xAQk4I/edit)

## 1. Model
It should look like a half human, half machine female android.
![qwa](https://user-images.githubusercontent.com/25332142/37762974-1dba756e-2dbe-11e8-8bab-7a8e89301d64.jpg)
Basically a nice mix between the two artworks you see here.
![asgasf](https://user-images.githubusercontent.com/25332142/37762978-1f2a8bb4-2dbe-11e8-9d62-4a742f4489e7.jpg)

## 2. Animations
In terms of animations you should be able to get a rough idea from the original abilties in HoN. 
Since the model is different the walking animations, attack animations, etc. should be properly adjusted and make sense. 

## 3. Icons
We need Icons for all his Abilities and Modifiers.

**Comment by warpdragon** [comment]
I claim this mother fucker

**Comment by warpdragon** [comment]
#2127

**Comment by warpdragon** [comment]
Some WIP pix:

![icons](https://user-images.githubusercontent.com/13878439/38143753-fb41528e-340f-11e8-81e2-397547ecffd0.png)

![portrait](https://user-images.githubusercontent.com/13878439/38143763-04c16722-3410-11e8-93ff-c2444226bd02.png)

![model](https://user-images.githubusercontent.com/13878439/38143771-0a22fa1e-3410-11e8-847b-ae4c868c0ed3.png)

![in engine](https://user-images.githubusercontent.com/13878439/38143785-10d7515c-3410-11e8-8917-882f66e31bcd.png)

**Comment by mikibun** [comment]
Can this be closed/Warpdragon be paid yet? @l34Um1
